frontend/src/pages/EmployeeProfile/EmployeeProfileContainer.tsx

```diff
@@ -8,7 +8,9 @@ import EmployeeProfile from './EmployeeProfile';
 
 const EmployeeProfileContainer: React.FC = () => {
   const params = useParams();
-  const { data: employeeResponse } = useGetEmployeeProfile(params.employeeId);
+  const { data: employeeResponse, refetch: refetchEmployeeProfile } = useGetEmployeeProfile(
```

Comment:
In general, we can do this with queryClient.refetchQueries([QUERY_KEYS.employeeProfile, employeeId]) at useAddCourseToEmployee mutation